src/dsp_tools/commands/excel2json/resources.py

```diff
@@ -118,7 +122,7 @@ def _row2resource(
     # load the cardinalities of this resource
     # if the excel sheet does not exist, pandas raises a ValueError
     try:
-        details_df = utl.read_and_clean_excel_file(excelfile=excelfile, sheetname=name)
+        details_df = read_and_clean_excel_file(excelfile=excelfile, sheetname=name)
     except ValueError as err:
         raise UserError(str(err)) from None
```

Comment:
I know this is not in scope of your PR - but this just screams for refactoring... :)

Reply:
My intention for this file is to add fail fast behaviour and implement the new error protocoll there. I started to do it in one PR but the diff got too big (https://github.com/dasch-swiss/dsp-tools/pull/663/files) and I will let this one die later.

For my next PR I am planning to change all this excel validation behaviour, this is why I did not touch any of the other errors but the validation.